tests/integration_tests/test_game.py

```diff
@@ -532,3 +537,241 @@ async def test_gamestate_ended_clears_references(
+        host_proto,
+        lambda cmd: cmd["command"] == "game_info" and cmd["launched_at"]
+    )
+    await host_proto.send_message({
+        "target": "game",
+        "command": "EnforceRating",
+        "args": []
+    })
+
+    # End the game
+    # Report results
+    for proto in (host_proto, guest_proto):
+        await proto.send_message({
+            "target": "game",
+            "command": "GameResult",
+            "args": [1, "victory 10"]
+        })
+        await proto.send_message({
+            "target": "game",
+            "command": "GameResult",
+            "args": [2, "recall defeat -5"]
+        })
+    # Report GameEnded
+    for proto in (host_proto, guest_proto):
+        await proto.send_message({
+            "target": "game",
+            "command": "GameEnded",
+            "args": []
+        })
+    # Now disconnect both players
+    for proto in (host_proto, guest_proto):
+        await proto.send_message({
+            "target": "game",
+            "command": "GameState",
+            "args": ["Ended"]
+        })
+
+    message = await asyncio.wait_for(mq_proto_all.read_message(), timeout=5)
+
+    assert message == {
+        "game_id": 41956,
+        "rating_type": "global",
+        "map_id": 7,
+        "featured_mod": "gw",
+        "sim_mod_ids": [],
+        "commander_kills": {},
+        "validity": "VALID",
+        "teams": [
+            {
+                "outcome": "DEFEAT",
+                "player_ids": [3],
+                "army_results": [[3, 2, "DEFEAT", ["recall"]]]
+            },
+            {
+                "outcome": "VICTORY",
+                "player_ids": [1],
+                "army_results": [[1, 1, "VICTORY", []]]
+            }
+        ]
+    }
+
+    with pytest.raises(asyncio.TimeoutError):
+        # We expect only one message to be broadcast
+        await asyncio.wait_for(mq_proto_all.read_message(), timeout=5)
+
+
+@pytest.mark.rabbitmq
+@fast_forward(30)
+async def test_galactic_war_2v1_game_ended_broadcasts_army_results(lobby_server, channel):
+    mq_proto_all = await connect_mq_consumer(
+        lobby_server,
+        channel,
+        "success.gameResults.create"
+    )
+
+    host_id, _, host_proto = await connect_and_sign_in(
+        ("test", "test_password"), lobby_server
+    )
+    guest_id, _, guest_proto = await connect_and_sign_in(
+        ("Rhiza", "puff_the_magic_dragon"), lobby_server
+    )
+    other_guest_id, _, other_guest_proto = await connect_and_sign_in(
+        ("newbie", "password"), lobby_server
+    )
+    await read_until_command(guest_proto, "game_info")
+    await read_until_command(other_guest_proto, "game_info")
+
+    # Set up the game
+    game_id = await host_game(host_proto, mod="gw")
+    await join_game(guest_proto, game_id)
+    await join_game(other_guest_proto, game_id)
+    # Set player options
+    await send_player_options(
+        host_proto,
+        [host_id, "Army", 1],
+        [host_id, "Team", 1],
```

Comment:
This is a 1v1v1 because 2 people are in the ffa team.

Reply:
Ah ok 🤦